socialhome/streams/app/components/Stream.vue

```diff
@@ -1,13 +1,13 @@
 <template>
     <div>
         <div class="container-flex">
-            <div id="new-content-container" class="hidden">
-                <a href="" id="new-content-load-link" onclick="return false;">
+            <div v-show="$store.state.stream.hasNewContent" id="new-content-container">
+                <b-button @click.prenvent.stop="onNewContentClick" class="new-content-load-link">
```

Comment:
I don't know yet. I very admit this is untested :)
I don't know well about HTML accessibility but a button or a link seem the way to go for visually disabled people. I should dig into Bootstrap 4 and VueBootstrap to see how components are rendered.